### PR TITLE
Publish 0.4.5 on npm. Update publish script.

### PR DESCRIPTION
**Pull request by djfarrelly**
### Purpose
To move our npm package versioning inline with the version of this repo and the python package.

### Notes
I accidentally published this on the beta channel 🙈 - For the next version we should publish on stable!

Using the node alpine image for a much smaller and faster to install image.

### CODE REVIEW

File: scripts/publish-node.sh [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+if [ $ARG == "beta" ]; then
+    COMMAND="npm publish --tag beta"
+else
+    COMMAND="npm publish"
+fi
+
+if [ ! -f $NPMRC_PATH ]; then
+    NPMRC_PATH="$PWD/.npmrc"
+    if [ ! -f $NPMRC_PATH ]; then
+        echo ".npmrc file not found, please run ./scripts/npm-login.sh"
```
Comment (michael-erasmus) [inline scripts/publish-node.sh]: Nice :smile_cat:

File: scripts/publish-node.sh [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+fi
+
+echo "Publishing @bufferapp/buda@$VERSION"
+echo "$COMMAND"
+
+docker run -it --rm \
+    -v $NPMRC_PATH:/root/.npmrc \
+    -v $PWD/packages/node:/app \
+    -w /app \
+    node:8.4-alpine \
```
Comment (djfarrelly) [inline scripts/publish-node.sh]: node alpine - 23mb
node (default) - 269mb

File: scripts/publish-node.sh [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+NPMRC_PATH="$HOME/.npmrc"
+
+# For beta release: ./scripts/publish-node.sh beta
+if [ $ARG == "beta" ]; then
+    COMMAND="npm publish --tag beta"
+else
+    COMMAND="npm publish"
+fi
+
+if [ ! -f $NPMRC_PATH ]; then
```
Comment (djfarrelly) [inline scripts/publish-node.sh]: First check the home fir then the local npmrc in this project dir

File: scripts/publish-node.sh (hunk 2 above)
Comment (michael-erasmus) [reply scripts/publish-node.sh]: Cool, maybe the npm-login task could use that tool

Comment (djfarrelly) [reply scripts/publish-node.sh]: Ah, I missed that, great call @michael-erasmus!

**Pull request merged**